osm2streets/src/intersection.rs

```diff
@@ -81,8 +81,29 @@ pub enum IntersectionControl {
     Construction,
 }
 
-/// The path that some group of adjacent lanes of traffic can take through an intersection.
-pub type Movement = (RoadID, RoadID);
+/// A manoeuvre that traffic can make through an intersection, from one road to another.
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct Turn {
+    pub from: RoadID,
+    pub to: RoadID,
+    pub via: IntersectionID,
+    // direction: TurnDirection,
+    /// The lane level movements, if they have been calculated.
+    pub movements: Option<Vec<TurnMovement>>,
+}
+
+/// A specific path that a vehicle can take through a turn.
+///
+/// Lane numbers are counted from the perspective of the direction of the turn.
```

Comment:
This stumped me for a few minutes; can we add an example or clarify the comment? LaneIDs normally index into `lanes_ltr`. Normally whatever direction we're thinking about a road, the lane ID is "absolute" -- 0 means the left side of the road based on its OSM-defined orientation. It feels like here we have a sudden explosion of possibilities, if we point from the right side of one road to the left side of another in the backwards direction, for instance...

And actually now I'm wondering if this refers to `LtrLaneNum` and not `LaneID` and am more confused :\